### PR TITLE
Feature/ins 537 inspection reproval screen

### PR DESCRIPTION
**Pull request by Leandro97**

### CODE REVIEW

File: StantUiIosLibrary/Classes/Inputs/PhotoInputView/PhotoCollectionViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        configureImageView(path: path)
+        configureCloseButton()
+    }
+    
+    func configureImageView(path: String) {
+        imageView           = UIImageView()
+        guard let imageView = imageView else { return }
+        
+        self.addSubview(imageView)
+        imageView.anchor(top: self.topAnchor,
```
Comment (vierrra) [inline StantUiIosLibrary/Classes/Inputs/PhotoInputView/PhotoCollectionViewCell.swift]: Indentation

Comment (Leandro97) [reply StantUiIosLibrary/Classes/Inputs/PhotoInputView/PhotoCollectionViewCell.swift]: Fixed

File: StantUiIosLibrary/Classes/Inputs/MultiLineTextView.swift [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        
+        let existingLines    = textView.text.components(separatedBy: CharacterSet.newlines)
+        let newLines         = text.components(separatedBy: CharacterSet.newlines)
+        let linesAfterChange = existingLines.count + newLines.count - 1
+        
+        if(text == "\n") {
+            return linesAfterChange <= textView.textContainer.maximumNumberOfLines
+        }
+
+        let newText              = (textView.text as NSString).replacingCharacters(in: range, with: text)
```
Comment (vierrra) [inline StantUiIosLibrary/Classes/Inputs/MultiLineTextView.swift]: Indentation

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        }
+
+        return cell
+    }
+    
+    public override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        if selectionType == .single {
+            selectedItems = []
+        } else if selectedItems.contains(indexPath.row) {
+            selectedItems                   = selectedItems.filter{ $0 != indexPath.row }
```
Comment (vierrra) [inline StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift:62]: Indentation of this block

Comment (vierrra) [inline StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Retract parameters

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            guard let cell                  = tableView.cellForRow(at: indexPath) as? SelectionTableViewCell else { return }
+            let radioButton                 = cell.accessoryView as? CustomRadioButton
+            radioButton?.deselect()
+            
+            return
+        }
+        
+        let cell                         = tableView.cellForRow(at: indexPath) as? SelectionTableViewCell
```
Comment (vierrra) [inline StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Retract parameters

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        let radioButton                  = cell?.accessoryView as? CustomRadioButton
+        radioButton?.select()
+        selectedItems.append(indexPath.row)
+    }
+    
+    public override func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
+        selectedItems                    = selectedItems.filter{ $0 != indexPath.row }
```
Comment (vierrra) [inline StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Retract parameters

File: StantUiIosLibrary/Classes/Inputs/MultiLineTextView.swift (hunk 2 above)
Comment (Leandro97) [reply StantUiIosLibrary/Classes/Inputs/MultiLineTextView.swift]: Fixed

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift (hunk 3 above)
Comment (Leandro97) [reply StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Fixed

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift (hunk 4 above)
Comment (Leandro97) [reply StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Fixed

File: StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift (hunk 5 above)
Comment (Leandro97) [reply StantUiIosLibrary/Classes/Tables/SelectionViewController/RadioButtonSelection/RadioButtonSelectionViewController.swift]: Fixed

**Review by vierrra — APPROVED** [review]

**Pull request merged**